sml_small/utils/pandas_wrapper.py

```diff
@@ -132,7 +132,7 @@ def run_thousand_pounds(
     target_variables_list = {}
     for value in target_variables_columns:
         if row[value] is None:
-            target_variables_list[value] = 'Nan'
+            target_variables_list[value] = float('Nan')
```

Comment:
Does not really matter but should these be NaN? Not Nan

Reply:
I have made them consistent throughout the code base now, there was a mix.